src/main/java/seedu/address/logic/commands/meeting/DeleteCommand.java

```diff
@@ -36,17 +36,13 @@ public DeleteCommand(Index targetIndex) {
     @Override
     public CommandResult execute(ModelMeeting modelMeeting) throws CommandException {
         requireNonNull(modelMeeting);
-        //TODO: Add getFilteredMeetingList
-        //List<Meeting> lastShownList = modelMeeting.getFilteredMeetingList();
-        List<Meeting> lastShownList = new ArrayList<Meeting>();
-
+        List<Meeting> lastShownList = modelMeeting.getFilteredMeetingList();
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
             throw new CommandException(MESSAGE_MEETING_DISPLAYED_INDEX);
```

Comment:
I think it would be great if we can use a more descriptive constant name to indicate an invalid index e.g. MESSAGE_MEETING_INDEX_INVALID

Reply:
I'm not sure but does the condition handle negative values or characters too?